index.html

```diff
@@ -1255,6 +1239,231 @@ <h3>
+          <p>
+            For example, the [=manifest/name=] member is a [=localizable
+            member=], and `name_localized` is its corresponding
+            [=manifest/*_localized=] member.
+          </p>
+        </aside>
+        <p>
+          A <dfn>language map</dfn> is an [=ordered map=] whose key is a
+          [=language tag=] and whose value is a [=localized value=]. The
+          <dfn>localized value</dfn> represents the value applied for the
+          [=language tag=] specified as the key.
+        </p>
+        <aside class="example" title="Localizing the application name">
+          <pre class="json">
+            {
+              "lang": "en-US",
+              "dir": "ltr",
+              "name": "Color Picker",
+              "name_localized": {
+                "de":    "Farbwähler",
+                "en":    {"value": "Color Picker"},
+                "en-GB": {"value": "Colour Picker", "dir": "ltr"},
+                "fr":    {"value": "Sélecteur de Couleur", "lang": "fr-CA", "dir": "ltr"},
+                "ar":    {"value": "منتقي الألوان", "dir": "rtl"}
+              }
+            }
+          </pre>
+        </aside>
+        <p>
+          [=manifest/*_localized=] members contain a [=language map=] that
+          defines [=localized values=] for the given [=localizable member=] in
+          the application. The user agent SHOULD use the user's localization
+          settings to select the [=localized value=] whose [=language tag=] key best
+          matches the user's preference. When no such [=localized value=] is
+          available, the default representation is used.
+        </p>
+        <section>
+          <h3>
+            Localizing text resources
+          </h3>
+          <p>
+            For [=manifest/*_localized=] members that accept a [=string=] as
+            their value, the [=localized value=] can either be a plain
+            localized [=string=] or an [=ordered map=] containing the following
+            members:
+          </p>
+          <dl>
+            <dt>
+              `value`
+            </dt>
+            <dd>
+              The localized [=string=].
+            </dd>
+            <dt>
+              `lang` (optional)
+            </dt>
+            <dd>
+              A [=language tag=].
+            </dd>
+            <dt>
+              `dir` (optional)
+            </dt>
+            <dd>
+              The [=text-direction=].
+            </dd>
+          </dl>
+          <p>
+            When a plain [=string=] is used, or when the `lang` or `dir`
+            members are omitted, `lang` falls back the [=language tag=] of the
+            [=language map=] key, and `dir` falls back to the [=manifest/dir=]
+            member of the [=manifest=].
+          </p>
+          <aside class="note">
+            <p>
+              The `dir` member must be specified if the direction of a
+              localized string differs from the default direction set in the
```

Comment:
We might need to formally define "default direction" and "default language"... or maybe call them "root language" and "root direction"? I don't mind which. 

 
```suggestion
              localized string differs from the [=default direction=] set in the
```